packages/core/tsconfig.json

```diff
@@ -14,6 +14,12 @@
     "target": "ES2015",
     "resolveJsonModule": true,
   },
+  "ts-node": {
```

Comment:
How would you have to run ts-node on this package?

I guess if you do this it will no need for this

```
pnpm run dev # on packages/core
npx ts-node ./xxx.ts # on examples
```